cub/cub/device/dispatch/tuning/tuning_scan_by_key.cuh

```diff
@@ -785,7 +984,28 @@ struct policy_hub
     using ScanByKeyPolicyT = decltype(select_agent_policy<sm90_tuning<key_t, ValueT, is_primitive_op<ScanOpT>()>>(0));
   };
 
-  using MaxPolicy = Policy900;
+  struct Policy1000 : ChainedPolicy<1000, Policy1000, Policy900>
+  {
+    // Use values from tuning if a specialization exists, otherwise pick Policy900
+    template <typename Tuning>
+    static auto select_agent_policy100(int)
+      -> AgentScanByKeyPolicy<Tuning::threads,
+                              Tuning::items,
+                              Tuning::load_algorithm,
+                              Tuning::load_modifier,
+                              BLOCK_SCAN_WARP_SCANS,
+                              Tuning::store_algorithm,
+                              typename Tuning::delay_constructor>;
+
+    template <typename Tuning>
+    // FIXME(bgruber): should we rather use `AccumT` instead of `ValueT` like the other default policies?
```

Comment:
nit: This comment seems unrelated here.
```suggestion
```

Reply:
I think I left this in for @gevtushenko to answer. @gevtushenko any action required here? Otherwise, I will just drop the comment.